platform-api/src/main/java/io/datacater/core/info/ResourceItem.java

```diff
@@ -0,0 +1,15 @@
+package io.datacater.core.info;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record ResourceItem(
+    @JsonProperty("url") String url, @JsonProperty("documentation-url") String documentationLink) {
```

Comment:
At other places, we use the camel case for JSON properties, e.g., [here](https://github.com/DataCater/datacater/blob/main/platform-api/src/main/java/io/datacater/core/stream/StreamEntity.java#L32). Let's stay consistent and rename it to `documentationUrl"? I would also suggest to keep the same name for the JSON property and the argument of the Java method.